docs/source/install.rst

```diff
@@ -32,6 +32,60 @@ Install from Source
     $ pip install git+https://github.com/pydata/pandas-gbq.git
 
 
+Install in Development Mode
+---------------------------
+
+Follow these instructions to install pandas-gbq if you want to make changes to
+the library.
+
+`Download the code <https://github.com/pydata/pandas-gbq/archive/master.zip>`__
+from GitHub or clone the git repository.
+
+.. code-block:: shell
+
+    $ git clone https://github.com/pydata/pandas-gbq.git
+    $ cd pandas-gbq
+
+Pip
+~~~
+
+To install pandas-gbq and its dependencies in development mode via pip
+
+.. code-block:: shell
+
+    $ pip install -e .
```

Comment:
You're right.

I'd like to keep these instructions for folks that aren't using conda. I've moved the section after and added a disclaimer that it's not necessary if using conda.